src/Microsoft.AspNetCore.Razor.Evolution/DefaultRazorIRLoweringPhase.cs

```diff
@@ -118,14 +118,44 @@ public override void VisitEndDirectiveBlock(DirectiveChunkGenerator chunkGenerat
 
             public override void VisitAddTagHelperSpan(AddTagHelperChunkGenerator chunkGenerator, Span span)
             {
+                _builder.Add(new DirectiveTokenIRNode()
+                {
+                    Content = chunkGenerator.LookupText,
+                    Descriptor = new DirectiveTokenDescriptor()
+                    {
+                        Kind = DirectiveTokenKind.String,
+                        Value = chunkGenerator.LookupText
```

Comment:
Don't set Value, it's used by the Literal `DirectiveTokenKind`. We should nuke the property entirely.